### DISCUSSION TITLE
Is there anything similar to "prefixed" for breakpoints?

### DISCUSSION THREAD

**Discussion by cesswhite** [unocss/unocss#686] [Q&A]
Personally I prefer to use the idea of desktop first then mobile, that's why the prefixed `"-"` before each utility breakpoint was very convenient for me, example `-sm:text-sm`

Does something similar already exist or are you planning to "review" this feature at some point?

**Comment by chu121su12** [comment]
Please elaborate what the `"-"` will do. Because currently, the token is used for negating rules that produces numbers.

**Reply by cesswhite** [reply]
Oh, I see... The `"-"` prefix for breakpoints acts like this

With prefix `"-"`, `@media` change to `"max-width"`
<img width="189" alt="Captura de Pantalla 2022-03-09 a la(s) 14 13 17" src="https://user-images.githubusercontent.com/44076866/157526857-3e800828-a1be-4343-ba00-4eefe2ed0e03.png">

Without prefix `"-"`, `@media` keep the `"min-width"`
<img width="189" alt="Captura de Pantalla 2022-03-09 a la(s) 14 13 41" src="https://user-images.githubusercontent.com/44076866/157527309-f53e6764-1420-4729-a794-13f1971358e1.png">

**Reply by chu121su12** [reply]
Currently the syntax is using `lt`: `lt-sm:text-sm`. You can use [preprocess](https://github.com/unocss/unocss#utilities-preprocess--prefixing) to achieve similar effect (see #529).
